com.unity.ml-agents/Runtime/Policy/RemotePolicy.cs

```diff
@@ -15,7 +15,7 @@ internal class RemotePolicy : IPolicy
         int m_AgentId;
         string m_FullyQualifiedBehaviorName;
 
-        protected ICommunicator m_Communicator;
+        internal ICommunicator m_Communicator;
```

Comment:
Hmm, maybe this doesn't matter if the whole class is internal now.

Reply:
seems ok